tests/test_converter.py

```diff
@@ -0,0 +1,39 @@
+from hypothesis import assume, given, reproduce_failure, seed
+from hypothesis.strategies import from_type, iterables, lists
+from pandas import pandas as pd
+from typing import Iterable, List, Sequence
+from tests import helpers
+
+from bankroll.model import Position, Trade
+from bankroll.converter import dataframeForModelObjects
+
+import unittest
+
+
+class TestModelConverter(unittest.TestCase):
+    @given(lists(from_type(Trade), min_size=3, max_size=3))
```

Comment:
I would recommend `min_size=0` for maximum testing, unless you have a specific reason why not.

```suggestion
    @given(lists(from_type(Trade), max_size=3))
```